scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/util/ChangeHttpConfiguration.java

```diff
@@ -350,11 +358,13 @@ private static void setCurrentConfiguration(String scheme, String port) {
             ChangeHttpConfiguration.port = isHttps ? WebProperties.WEB_HTTPS_PORT.getValueAsInt()
                                                    : WebProperties.WEB_HTTP_PORT.getValueAsInt();
         }
+        System.out.println("Applying configuration: scheme=" + (isHttps ? "https" : "http") + " hostname=" +
+                           (hostname != null ? hostname : "<unchanged>") + " port=" + ChangeHttpConfiguration.port);
```

Comment:
we cannot extract the hostname when hostname is null ?

Reply:
I can try, but it is annoying as it changes the organization of the code. I could also display this message at the end which allows not to change the organization

Reply:
@mboussaa I've done the requested improvement